tests/README.md

```diff
@@ -15,9 +15,9 @@ for the current code in the users ufs-srweather-app directory.  It consists of t
 
 Currently, the following configurations are supported:
 
-Machine     | Derecho | Cheyenne    | Hera   | Jet    | Orion  | wcoss2  |
-------------|---------|-------------|--------|--------|--------|---------|
-Compiler(s) | Intel   | Intel, GNU  | Intel  | Intel  | Intel  | Intel   |
+Machine     | Derecho | Ursa   | Hera   | Jet    | Orion  | wcoss2  |
+------------|---------|--------|--------|--------|--------|---------|
+Compiler(s) | Intel   | Intel  | Intel  | Intel  | Intel  | Intel   |
```

Comment:
I see Ursa GNU module files and WE2E tests with this PR. Are we not officially supporting GNU builds on Ursa?